src/components/pages/account/AssetInfo.tsx

```diff
@@ -13,9 +15,12 @@ export type AssetInfoProps = {
 export default function AssetInfo({
   account,
 }: AssetInfoProps): JSX.Element {
-  const { data: t } = useAccountTotal(account.guid);
+  const { data: t0 } = useAccountsTotals();
+  const { data: t1 } = useAccountsTotals(DateTime.now().minus({ month: 1 }).endOf('month'));
```

Comment:
End of month previous total to make the difference between today and last month (increase or decrease)